python/IECoreArnoldTest/RendererTest.py

```diff
@@ -1888,7 +1902,7 @@ def testOSLShaders( self ) :
 			n = arnold.AiNodeLookUpByName( universe, "testPlane" )
 
 			add = arnold.AiNodeGetPtr( n, "shader" )
```

Comment:
`add` is a bit misleading now this is actually a switch.

Reply:
Addressed in place